events/ready.js

```diff
@@ -1,21 +1,4 @@
 module.exports = async client => {
-  // Why await here? Because the ready event isn't actually ready, sometimes
```

Comment:
Legitimately you shouldn't be attempting to pull bot information if the bot isn't ready, undo this change.

Reply:
changed back to previous in next commit